frontend/src/pages/_document.tsx

```diff
@@ -15,6 +15,12 @@ export default class MyDocument extends Document {
         <Head>
           <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" />
           <link rel="icon" type="image/png" href="/images/favicon.png" />
+
+          <link
+            // eslint-disable-next-line max-len
+            href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700|Source+Code+Pro:400,700"
```

Comment:
Let's follow the standard of fonts, you can convert your ttf fonts in https://www.fontsquirrel.com/tools/webfont-generator and then upload the fonts to /frontend/public/fonts, and then register the font at our theme.tsx file like this:

```css
  @font-face {
    font-family: 'Open Sans';
    src: url('/fonts/OpenSans/font.woff2') format('woff2'),
        url('/fonts/OpenSans/font.woff') format('woff');
    font-weight: normal;
    font-style: normal;
  }

  @font-face {
    font-family: 'Source Code Pro';
    src: url('/fonts/SourceCodePro/font.woff2') format('woff2'),
        url('/fonts/SourceCodePro/font.woff') format('woff');
    font-weight: normal;
    font-style: normal;
  }
```